argv-t.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

Reply:
Unless you're not using any bash-specific features, in which case, just `#!/bin/sh`

Reply:
Ahh that change got left behind on another branch. I appreciate the reminder. I will change that. At this time that script is not being used by the pipeline and can only be used by linux users. Should it be removed all together and I will simply keep a local copy?

Reply:
It is probably wise to keep it available for the moment.